### ISSUE TITLE
update mailgun-js version - as it uses a vulnerable debug version

### ISSUE THREAD

**Issue by miacobv** [log4js-node/log4js-node#630]
``` 
           │ Regular Expression Denial of Service                               │
├────────────┼────────────────────────────────────────────────────────────────────┤
│ Name       │ debug                                                              │
├────────────┼────────────────────────────────────────────────────────────────────┤
│ CVSS       │ 3.7 (Low)                                                          │
├────────────┼────────────────────────────────────────────────────────────────────┤
│ Installed  │ 2.2.0                                                              │
├────────────┼────────────────────────────────────────────────────────────────────┤
│ Vulnerable │ <= 2.6.8 || >= 3.0.0 <= 3.0.1                                      │
├────────────┼────────────────────────────────────────────────────────────────────┤
│ Patched    │ >= 2.6.9 < 3.0.0 || >= 3.1.0                                       │
├────────────┼────────────────────────────────────────────────────────────────────┤
│ Path       │  log4js@2.4.1 > mailgun-js@0.7.15 >    │
│            │ debug@2.2.0                                                        │
├────────────┼────────────────────────────────────────────────────────────────────┤
│ More Info  │ https://nodesecurity.io/advisories/534       
```
https://github.com/log4js-node/log4js-node/blob/master/package.json#L63

Change to mailgun-js 0.13.1 as this one uses a good version of debug

 https://github.com/bojand/mailgun-js/blob/master/package.json#L24

**Comment by nomiddlename** [comment]
Will fix this as part of #597

**Comment by nomiddlename** [comment]
Updated mailgun-js to version 0.18.0 in log4js@2.6.0
